extension/sparkmonitor/kernelextension.py

```diff
@@ -209,9 +209,9 @@ def configure(conf):
     logger.info(os.environ['SPARKMONITOR_KERNEL_PORT'])
     conf.set('spark.extraListeners',
              'sparkmonitor.listener.JupyterSparkMonitorListener')
-    jarpath = os.path.abspath(os.path.dirname(__file__)) + '/listener.jar'
-    # import pkg_resources
-    # jarpath = pkg_resources.resource_filename(__name__, "/listener.jar")
+    # jarpath = os.path.abspath(os.path.dirname(__file__)) + '/listener.jar'
+    import pkg_resources
+    jarpath = pkg_resources.resource_filename(__name__, '/listener.jar')
```

Comment:
If there isn't a reason for importing this inline, add it to imports at top of file.

Reply:
Fixed